tfjs-backend-webgpu/src/setup_test.ts

```diff
@@ -115,6 +115,9 @@ const TEST_FILTERS: TestFilter[] = [
       'leakyrelu'           // Not yet implemented.
```

Comment:
Please remove `leakyrelu` from `excludes` of `relu`. And change `include: 'relu'` -> `startsWith: 'relu'`. Otherwise, `leakyrelu` is not really tested. Similarly, change `include: 'elu'` -> `startsWith: 'elu'`. There may be some other failures due to this change. You can just exclude those cases in the corresponding category and add the failure reason.

Reply:
Checked all the test case of leakyRelu  and exclude one failure case.

However  some other failure blocked at the beginning, cause `dot(vecN<i32>, vecN<i32>)` (https://github.com/tensorflow/tfjs/blob/master/tfjs-backend-webgpu/src/shader_preprocessor.ts#L272)  not supported ( find it recently added [here](https://github.com/gpuweb/gpuweb/issues/2231)) and it lead to wgsl parse error, maybe it's not supported for Metal.
And I temporarily remove them and successfully executed.

Reply:
I still meet some errors for `fused` operators with `gradient` tests. You need to add `'gradient',   // Kernel 'Step' not registered`  in `excludes` part of `include: 'fused',`. Sorry that it's a mess to  use `includes` at the beginning. We properly need a refactor for the test filtering later.

Are you using the latest chrome canary in Mac? If `dot(vecN<i32>, vecN<i32>)` issue still exists in latest chrome, we need to report a bug to tint to support.

Reply:
Sorry that the dot(vecN<i32>, vecN<i32>) issue exists in chrome 96.
(Additionally, Chrome has started webgpu trial in Chrome 94，wondering weather the compatibility should be considered.)

And I've tried chrome 98 canary and update it to the latest just now and try `yarn test` again, but there still many other failure block the test, seems like fail error aborted sometimes and lead to webgpu backend initialize fail, maybe I missing something.

As for the errors for `fused` operators with `gradient` tests，wondering if they are associated with the leakyRelu ops, cause the tests cases for `fused` who are including `leakyrelu` executed successfully in my env.

Reply:
@justmywyw I haven't tried Mac's `yarn test` situation previously. But on windows, we have 100% pass rate with current filter rule. I will meet below error after applying your PR:
```
FAILED fused matmul webgpu {"WEBGPU_CPU_FORWARD":false} fused A x B with relu bias gradient
debug.js:21 Error: Kernel 'Step' not registered for backend 'webgpu'
FAILED fused matmul webgpu {"WEBGPU_CPU_FORWARD":false} gradient with clones A x B with relu
Error: Kernel 'Step' not registered for backend 'webgpu'
FAILED fused matmul webgpu {"WEBGPU_CPU_FORWARD":false} fused matmul with relu6 and gradients
Error: Kernel 'Step' not registered for backend 'webgpu'
FAILED fused matmul webgpu {"WEBGPU_CPU_FORWARD":false} fused A x B with relu and broadcasted bias gradient
Error: Kernel 'Step' not registered for backend 'webgpu'
FAILED fused matmul webgpu {"WEBGPU_CPU_FORWARD":false} fused A x B with relu gradient
Error: Kernel 'Step' not registered for backend 'webgpu'
```
I think the error is caused by changing `include: 'relu'` to `startsWith: 'relu'`. So please add `'gradient', // Kernel 'Step' not registered` in `excludes` part of `include: 'fused'` so that all cases can pass on windows. After that, I will merge your PR, thanks.

I will check the situations on mac. Thanks for your report for the mac issue.

Reply:
>there still many other failure block the test, seems like fatal error aborted sometimes and lead to webgpu backend initialize fail, maybe I missing something.

fyi, I just tried the latest chrome canary 99.0.4764.2 on Mac. I didn't meet the error that webgpu backend initialization fails. There are totally 3 failed cases (2 resizeNearestNeighbor cases and one image.transform case) in master branch. You can ignore these 3 cases for now.

Reply:
@qjia7 I add the exclude of `gradient` as your test result said.

However, I tried the latest chrome canary 99.0.4764.2 on my Mac, run `yarn test` on a new cloned tfjs project's master branch and receive the error below : 
````
Chrome 99.0.4764.2 (Mac OS 10.15.7): Executed 32 of 4128 (20 FAILED) (0 secs / 0Chrome 99.0.4764.2 (Mac OS 10.15.7) less webgpu {"WEBGPU_CPU_FORWARD":false} broadcasting Tensor4D shapes - float32 FAILED
	AbortError: Device is lost

Chrome 99.0.4764.2 (Mac OS 10.15.7) fused depthwiseConv2D webgpu {"WEBGPU_CPU_FORWARD":false} throws when input is int32 FAILED
	Error: The highest priority backend 'webgpu' has not yet been initialized. Make sure to await tf.ready() or await tf.setBackend() before calling other methods

````
It must be something wrong with my environment, and I‘ll try to figure out later and post here.

Reply:
Are you running chrome canary with flag `--enable-unsafe-webgpu`?

Reply:
The flag `--enable-unsafe-webgpu` is added by karma, and I've checked the console that `GPU` exists on window.
And the test case fail after some success executed ( which means webgpu have been  initialized at the beginning), maybe some fatal error lead to gpu device lost.

Reply:
Ah, you are right. The flag --enable-unsafe-webgpu is added by karma. I don't know the device lost reason. Maybe try another device to see if you can reproduce it. Please file a bug to us and attach your hardware information to see if we can reproduce it in our side.